scripts/walrus-store-sliced.sh

```diff
@@ -0,0 +1,154 @@
+#!/bin/bash
```

Comment:
question: Is this a utility script just for testing purpose?

Reply:
Yes, it's primarily for testing and just as an example illustrating a basic splitting flow. It does not currently handle gathering the blob IDs in an elegant way. https://github.com/MystenLabs/walrus/pull/2637 is intended to be a more elegant solution to that, but still requires some work.